launch/kinect2_bridge_cpu.launch

```diff
@@ -0,0 +1,85 @@
+<launch>
```

Comment:
What's the purpose of having `kinect2_bridge.launch` and `kincet_bridge_cpu.launch`?  Can we put a short description here and add instruction to README?